schemas/com.snowplowanalytics.snowplow.badrows/enrichment_failures/jsonschema/1-0-0

```diff
@@ -0,0 +1,309 @@
+                        "description": "Field which did not meet the enrichment's expectations",
+                        "maxLength": 128
+                      },
+                      "value": {
+                        "type": [ "string", "null" ],
+                        "description": "Stringified representation of the value which did not meet the enrichment's expectations"
+                      },
+                      "expectation": {
+                        "type": "string",
+                        "description": "Description of the expectation which was not met"
+                      }
+                    },
+                    "required": ["field", "value", "expectation" ],
+                    "additionalProperties": false
+                  },
+                  {
+                    "type": "object",
+                    "description": "Error which was internal to the enrichment regarding the enrichment's output data, this is a schema validation error",
+                    "properties": {
+                      "errors": {
+                        "type": "array",
+                        "minItems": 1,
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "path": {
+                              "type": "string",
+                              "description": "JSON Path to an issue in the faulty JSON Schema"
+                            },
+                            "message": {
+                              "type": "string",
+                              "description": "Human-readable message describing the issue with the schema"
+                            }
+                          },
+                          "required": [ "path", "message" ],
+                          "additionalProperties": false
+                        }
+                      }
+                    },
+                    "required": [ "errors" ],
+                    "additionalProperties": false
+                  },
+                  {
+                    "type": "object",
+                    "description": "Error which was external to the enrichment, e.g. a connection issue",
+                    "properties": {
+                      "error": {
+                        "type": "string",
+                        "description": "Error that occurred"
+                      }
+                    },
+                    "required": [ "error" ],
+                    "additionalProperties": false
+                  }
+                ]
+              }
+            },
+            "required": [ "enrichment", "message" ],
+            "additionalProperties": false
+          }
+        }
+      }
+    },
+    "payload": {
+      "type": "object",
+      "description": "The payload that resulted in enrichment failures",
+      "properties": {
+        "trackerProtocolPayload": {
+          "type": "object",
```

Comment:
Once an event is through to enrichment, validation has been completed. Wouldn't it then be possible to make the `unstruct_event` and `contexts` fields available for querying? There might be useful information about e.g. mobile app version that is buried in a mobile-specific context, for example.

Reply:
true, I think I missed those, will add them